### ISSUE TITLE
running inside a vpc

### ISSUE THREAD

**Issue by zeph** [goadapp/goad#100]
[testing aha.io integration] 

we want to be able to run goad within a VPC for testing non-internet-exposed staging/testing environments

**Comment by tyler-mauthe-hs** [comment]
Is this being worked on? I can take it on if not! Looks pretty straightforward...

**Comment by gserra-olx** [comment]
@tyler-mauthe-hs feel free to take it over... but...

I'm currently asking the consulting company which is working on GOAD for us at OLX to draw a line about extra parameters on the cli... therefore... they'll ensure that we have a 1:1 map with the Apache Benchmark CLI and drop any extra parameter... with the exception of a single one that will be a boolean switch true/false to init the local working folder (a bit like ruby on rails or any other framework that initializes a boiler plate to start from) 

any new extra parameter will be just exposed in an INI file for configuration

so, maybe you wanna wait that they do that and I release a new version, before starting to implement and get to rework afterwards...

**Comment by gserra-olx** [comment]
@cwaltken-edrans this parameter is currently not in the INI created by v`2.0.1`

**Comment by cwaltken-edrans** [comment]
That's correct. Goad still does not have features that allow configuring the aws infrastructure setup in more detail.
